src/styles/blocks/catalog.scss

```
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: color 300ms, transform 300ms;
```

Dont forget about variables)
in production - we always have a variable for transition)
```suggestion
    transition: color $transitionDuration, transform $transitionDuration;
```